.gitpod.yml

```diff
@@ -6,6 +6,9 @@ ports:
     onOpen: ignore
 
 tasks:
```

Comment:
I think `pre-commit` installation can be done in dockerfile than gitpod task

Reply:
I think I was initially trying to install it from dockerfile but somehow it wasn't there when the workspace launched, maybe because it was installed under `/workspace`? After that point, I head back to installing it from gitpod tasks.

Reply:
pre-commit, ideally, would be done as a [pre-commit hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks#:~:text=The%20pre%2Dcommit,on%20new%20methods.) on the client-side. 

Users can still override:
```bash
git commit . -m 'quick fix' --no-verify
```

But our Git Action would fail the build (like it does now) if pre-commit step fails.

Reply:
@kylos101 `pre-commit install` installs the git hooks.[^1]

[^1]: https://pre-commit.com/#3-install-the-git-hook-scripts